cmd/template/dbdriver/files/env/scylla.tmpl

```diff
@@ -0,0 +1,12 @@
+{{- if .AdvancedOptions.docker }}
+BLUEPRINT_DB_HOSTS=scylla_bp
+{{- else }}
+BLUEPRINT_DB_HOSTS=localhost
+{{- end }}
+BLUEPRINT_DB_PORT=9042
+BLUEPRINT_DB_CONSISTENCY="LocalQuorum"
+# BLUEPRINT_DB_KEYSPACE=
+# BLUEPRINT_DB_USERNAME=danielhe4rt
```

Comment:
Probably better clue of some usual default user like `cassandra` instead

Reply:
My bad. I was testing it locally and forgot.

Just sent a fix.